from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Tela_Comparacao(object):
    """

        Classe que contem todas as configurações da tela de comparacao da ferramenta.

        Nessa classe usa-se propriedades da biblioteca PyQt5 para facilitar o
        gerenciamento das configurações e interações da tela em questão.

    """
    def setupUi(self, Tela_Comparacao):
        Tela_Comparacao.setObjectName("Tela_Comparacao")
        Tela_Comparacao.resize(922, 586)
        self.centralwidget = QtWidgets.QWidget(Tela_Comparacao)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setObjectName("gridLayout")
        self.verticalLayout_3 = QtWidgets.QVBoxLayout()
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setObjectName("label_3")
        self.horizontalLayout.addWidget(self.label_3)
        self.comboBox = QtWidgets.QComboBox(self.centralwidget)
        self.comboBox.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.comboBox.setObjectName("comboBox")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.horizontalLayout.addWidget(self.comboBox)
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setObjectName("label_4")
        self.horizontalLayout.addWidget(self.label_4)
        self.match = QtWidgets.QLineEdit(self.centralwidget)
        self.match.setClearButtonEnabled(True)
        self.match.setObjectName("match")
        self.horizontalLayout.addWidget(self.match)
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setObjectName("label_5")
        self.horizontalLayout.addWidget(self.label_5)
        self.mismatch = QtWidgets.QLineEdit(self.centralwidget)
        self.mismatch.setClearButtonEnabled(True)
        self.mismatch.setObjectName("mismatch")
        self.horizontalLayout.addWidget(self.mismatch)
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setObjectName("label_6")
        self.horizontalLayout.addWidget(self.label_6)
        self.gap = QtWidgets.QLineEdit(self.centralwidget)
        self.gap.setClearButtonEnabled(True)
        self.gap.setObjectName("gap")
        self.horizontalLayout.addWidget(self.gap)
        self.verticalLayout_3.addLayout(self.horizontalLayout)
        self.line_3 = QtWidgets.QFrame(self.centralwidget)
        self.line_3.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_3.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_3.setObjectName("line_3")
        self.verticalLayout_3.addWidget(self.line_3)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setObjectName("verticalLayout")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setObjectName("label")
        self.verticalLayout.addWidget(self.label)
        self.setar_sequencia1 = QtWidgets.QTextEdit(self.centralwidget)
        self.setar_sequencia1.setReadOnly(True)
        self.setar_sequencia1.setObjectName("setar_sequencia1")
        self.verticalLayout.addWidget(self.setar_sequencia1)
        self.botao_abrir_arquivo1 = QtWidgets.QPushButton(self.centralwidget)
        self.botao_abrir_arquivo1.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.botao_abrir_arquivo1.setObjectName("botao_abrir_arquivo1")
        self.verticalLayout.addWidget(self.botao_abrir_arquivo1)
        self.horizontalLayout_3.addLayout(self.verticalLayout)
        self.line = QtWidgets.QFrame(self.centralwidget)
        self.line.setFrameShape(QtWidgets.QFrame.VLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line.setObjectName("line")
        self.horizontalLayout_3.addWidget(self.line)
        self.verticalLayout_2 = QtWidgets.QVBoxLayout()
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setObjectName("label_2")
        self.verticalLayout_2.addWidget(self.label_2)
        self.setar_sequencia2 = QtWidgets.QTextEdit(self.centralwidget)
        self.setar_sequencia2.setReadOnly(True)
        self.setar_sequencia2.setObjectName("setar_sequencia2")
        self.verticalLayout_2.addWidget(self.setar_sequencia2)
        self.botao_abrir_arquivo2 = QtWidgets.QPushButton(self.centralwidget)
        self.botao_abrir_arquivo2.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.botao_abrir_arquivo2.setObjectName("botao_abrir_arquivo2")
        self.verticalLayout_2.addWidget(self.botao_abrir_arquivo2)
        self.horizontalLayout_3.addLayout(self.verticalLayout_2)
        self.verticalLayout_3.addLayout(self.horizontalLayout_3)
        self.line_2 = QtWidgets.QFrame(self.centralwidget)
        self.line_2.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_2.setObjectName("line_2")
        self.verticalLayout_3.addWidget(self.line_2)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.botao_voltar = QtWidgets.QPushButton(self.centralwidget)
        self.botao_voltar.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../images/voltar.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.botao_voltar.setIcon(icon)
        self.botao_voltar.setObjectName("botao_voltar")
        self.horizontalLayout_2.addWidget(self.botao_voltar)
        self.botao_realizar_comparacao = QtWidgets.QPushButton(self.centralwidget)
        self.botao_realizar_comparacao.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.botao_realizar_comparacao.setObjectName("botao_realizar_comparacao")
        self.horizontalLayout_2.addWidget(self.botao_realizar_comparacao)
        self.verticalLayout_3.addLayout(self.horizontalLayout_2)
        self.gridLayout.addLayout(self.verticalLayout_3, 1, 0, 1, 1)
        self.line_4 = QtWidgets.QFrame(self.centralwidget)
        self.line_4.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_4.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_4.setObjectName("line_4")
        self.gridLayout.addWidget(self.line_4, 0, 0, 1, 1)
        Tela_Comparacao.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(Tela_Comparacao)
        self.statusbar.setObjectName("statusbar")
        Tela_Comparacao.setStatusBar(self.statusbar)

        self.retranslateUi(Tela_Comparacao)
        QtCore.QMetaObject.connectSlotsByName(Tela_Comparacao)

    def retranslateUi(self, Tela_Comparacao):
        _translate = QtCore.QCoreApplication.translate
        Tela_Comparacao.setWindowTitle(_translate("Tela_Comparacao", "Tela de Comparações - GeMapCom"))
        self.label_3.setText(_translate("Tela_Comparacao", "TIPO DE COMPARAÇÃO:"))
        self.comboBox.setItemText(0, _translate("Tela_Comparacao", "GLOBAL"))
        self.comboBox.setItemText(1, _translate("Tela_Comparacao", "LOCAL"))
        self.label_4.setText(_translate("Tela_Comparacao", "MATCH:"))
        self.label_5.setText(_translate("Tela_Comparacao", "MISMATCH:"))
        self.label_6.setText(_translate("Tela_Comparacao", "GAP:"))
        self.label.setText(_translate("Tela_Comparacao", "SEQUÊNCIA 1:"))
        self.botao_abrir_arquivo1.setText(_translate("Tela_Comparacao", "ABRIR ARQUIVO"))
        self.label_2.setText(_translate("Tela_Comparacao", "SEQUÊNCIA 2:"))
        self.botao_abrir_arquivo2.setText(_translate("Tela_Comparacao", "ABRIR ARQUIVO"))
        self.botao_voltar.setText(_translate("Tela_Comparacao", "VOLTAR"))
        self.botao_realizar_comparacao.setText(_translate("Tela_Comparacao", "RELIZAR COMPARAÇÃO"))
